src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -471,11 +471,18 @@ private void BeforeAttribute()
                 return;
             }
 
+            // Minimized attribute
             if (!At(HtmlSymbolType.Equals))
             {
-                // Saw a space or newline after the name, so just skip this attribute and continue around the loop
```

Comment:
Most of the comment at this spot in previous iterations was useful.  It described what the `Output()` calls would do.  My complaint was about the last ("in most cases") bit.